music/live-coding-ksql-music.adoc

```diff
@@ -60,39 +60,39 @@ The Docker compose file includes a container that auto-generates source data to
 * `play-events` : stream of play events (“song X was played”)
 * `song-feed` : stream of song metadata (“song X was written by artist Y”)
 
-image:images/ksql-music-demo-source-data.jpg[source-data]
+image::images/ksql-music-demo-source-data.jpg[source-data]
 
-After you bring up the demo, use Google Chrome to navigate to http://localhost:9021[Confluent Control Center]. Under `MANAGEMENT` select `Topics`, then scroll down and click on the topic and select `Inspect`.
+After you bring up the demo, use a web browser to navigate to http://localhost:9021[Confluent Control Center]. Select the `controlcenter.cluster` Cluster, then select `Topics`.  Find the topic and select `Inspect`.
```

Comment:
Not sure about our compatibility story: I think we run okay on Firefox but I don't know which browsers we support officially.